web/src/main/webResources/WEB-INF/config.properties

```diff
@@ -42,3 +42,7 @@ urlChecker.UserAgent=GeoNetwork Link Checker
 
 map.bbox.background.service=https://ows.terrestris.de/osm/service?SERVICE=WMS&amp;REQUEST=GetMap&amp;VERSION=1.1.0&amp;LAYERS=OSM-WMS&amp;STYLES=default&amp;SRS={srs}&amp;BBOX={minx},{miny},{maxx},{maxy}&amp;WIDTH={width}&amp;HEIGHT={height}&amp;FORMAT=image/png
 
+# If false, the services to draw map extents (region.getmap and {metadatauuid}/extents.png) accept urls for
+# map services to provide the background layers. Otherwise the only allowed options are from the settings configuration
+# or a named bg layer from regionGetMapBackgroundLayers
+metadata.extentApi.disableFullUrlBackgroundMapServices=false
```

Comment:
Maybe better to set it to true by default (most users are using the internal service). It will make the default installation more secured.